collect_app/src/main/java/org/odk/collect/android/utilities/ZipUtils.java

```diff
@@ -58,33 +58,12 @@ public static void unzip(File[] zipFiles) {
         }
     }
 
-    public static File extractFirstZipEntry(File zipFile, boolean deleteAfterUnzip)
-            throws IOException {
-        ZipInputStream zipInputStream = null;
-        File targetFile = null;
-        try {
-            zipInputStream = new ZipInputStream(new FileInputStream(zipFile));
-            ZipEntry zipEntry = zipInputStream.getNextEntry();
-            if (zipEntry != null) {
-                targetFile = doExtractInTheSameFolder(zipFile, zipInputStream, zipEntry);
-            }
-        } finally {
-            IOUtils.closeQuietly(zipInputStream);
-        }
-
-        if (deleteAfterUnzip && targetFile != null && targetFile.exists()) {
-            FileUtils.deleteAndReport(zipFile);
-        }
-
-        return targetFile;
-    }
-
     private static File doExtractInTheSameFolder(File zipFile, ZipInputStream zipInputStream,
```

Comment:
The return value of this method is never being used when called, so this can be made to return `void`